purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/BillingClientUseCase.kt

```diff
@@ -20,6 +20,7 @@ internal typealias ExecuteRequestOnUIThreadFunction = (delayInMillis: Long, onEr
 private const val MAX_RETRIES_DEFAULT = 3
 private const val RETRY_TIMER_START_MILLISECONDS = 878L // So it gets close to 15 minutes in last retry
 internal const val RETRY_TIMER_MAX_TIME_MILLISECONDS = 1000L * 60L * 15L // 15 minutes
+internal const val RETRY_TIMER_MAX_TIME_MILLISECONDS_FOREGROUND = 1000L * 4L // 4 seconds
```

Comment:
This gives for 3 retries and up to ~6s delay, which I think is more than enough. We should fail if it's failing more than that I think.